src/components/Toolbar.js

```diff
@@ -10,13 +11,64 @@ const shortcutIds = [
   'search',
 ]
 
-export const Toolbar = connect(({ settings: { dark } }) => ({ dark }))(({ dark }) =>
-  <div className="toolbar">
-    {shortcutIds.map(id => {
-      const { name, svg: Icon, exec } = shortcutById(id)
-      return <div key={id} className="toolbar-icon" title={name} onClick={exec}>
-        <Icon fill={dark ? 'white' : 'black'} />
-      </div>
-    })}
-  </div>
-)
+const ToolbarMessageOverlay = ({ name, description }) => {
+    return (
+        <div className={'toolbar-overlay'}>
+            <div className={'overlay-name'}>{ name }</div>
+            <div className={'overlay-body'}>{ description }</div>
+        </div>
+    )
+}
+
+var holdTimer
+var currentId
+
+const clickOrHoldAction = (id, name, description) => {
+    currentId = id
+    // on chrome setTimeout doesn't seem to work on the first click, clearing it before hand fixes the problem
+    clearTimeout(holdTimer)
+
+    holdTimer = setTimeout(() => {
+        const info = {
+            id,
+            name,
+            description
+        }
+        overlayReveal(info)
```

Comment:
Avoid single-use variables when you can inline them.

Reply:
Ok